js/db.js

```diff
@@ -756,6 +756,93 @@ define(["dao", "globals", "lib/bluebird", "lib/davis", "lib/underscore", "util/h
                     playerFeatStore.createIndex("tid", "tid", {unique: false});
                 }());
             }
+            if(event.oldVersion <= 13) {
```

Comment:
Just made all requested changes (ESLint too!), updated current DB version to 15 in connectLeague(). Will push to this branch in a while